docs/doxygen/mainpages/platdetails.h

```diff
@@ -180,13 +175,13 @@ used by wxWidgets to e.g. use toolkit-specific features.
 In such case (or when you want to e.g. write a port-specific patch) it can be
 necessary to use the underlying toolkit API directly:
 
-- wxMSW port uses win32 API: see MSDN docs at http://msdn2.microsoft.com/en-us/library/ms649779.aspx
+- wxMSW port uses Win32 API: see MSDN docs at https://msdn2.microsoft.com/en-us/library/ms649779.aspx
 - wxGTK port uses GTK+ and other lower-level libraries; see
-  - GTK+ docs at http://library.gnome.org/devel/gtk/unstable/
-  - GDK docs at http://library.gnome.org/devel/gdk/unstable/
-  - GLib docs at http://library.gnome.org/devel/glib/unstable/
-  - GObject docs at http://library.gnome.org/devel/gobject/unstable/
-  - Pango docs at http://library.gnome.org/devel/pango/unstable/
-- wxOSX port uses the Cocoa API: see Cocoa docs at http://developer.apple.com/cocoa
+  - GTK+ docs at https://library.gnome.org/devel/gtk/unstable/
```

Comment:
Also here I'm not sure if we should be linking to `unstable`. `https://developer.gnome.org/gtk/` seems to redirect to the appropriate GTK version (3 right now).